scenario03/resources/user.properties

```diff
@@ -46,3 +46,16 @@ secondaryJDBCDomain1=SECONDARYJDBC1
 secondaryJDBCDomain2=SECONDARYJDBC2
 fbIncorrectClientId=abcd34567
 isCallbackHost=is.qa.com
+
+#UserStores
+
+secondaryUS1_url=jdbc:mysql://192.168.48.231:3306/IS_Scenario03_Userstore1?useSSL=false
```

Comment:
This should be parameterized with Mysql host and port as it will allow to run on any type of infrastructure.

Reply:
Modified accordingly to export custom database host and port of secondary user stores.